tools/mo/openvino/tools/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -258,10 +258,7 @@ def serialize_runtime_info(node, parent_element: Element):
         attribute = SubElement(rt_info, 'attribute')
         attribute.set('name', name)
         attribute.set('version', str(version))
-        params = info_elem.serialize(node)
-        if len(params) == 0:
-            rt_info.remove(attribute)
-            continue
+        params = info_elem.serialize(node) if not isinstance(info_elem, dict) else info_elem
```

Comment:
Is it impossible empty params now? Possibly add assert for this case.

Reply:
The check was removed because now we need to support save attributes only with name and version, without value.
For example:
			<rt_info>
				<attribute name="decompression" version="0"/>
				<attribute name="fused_names" version="0" value="data_mul_646711813"/>
			</rt_info>